core/Cargo.toml

```diff
@@ -21,7 +21,7 @@ std = ["math/std", "winter-utils/std", "miden-crypto/std"]
 
 [dependencies]
 math = { package = "winter-math", version = "0.6", default-features = false }
-miden-crypto = { package = "miden-crypto", git = "https://github.com/0xPolygonMiden/crypto.git", branch = "next", default-features = false }
+miden-crypto = {git = "https://github.com/0xPolygonMiden/crypto", branch = "al-falcon-decorator-tmp" }
```

Comment:
I think this can be reverted back to the `next` branch.

Reply:
Yes, but I am getting an error due to incompatible versions of Blake3

Reply:
Interesting! For some reason I'm not getting these errors on #1084 - let me dig into this.

Reply:
Interesting!
Thank you!

Reply:
I think I might have fixed it - take a look at my latest commit.